spec/lib/appsignal/config_spec.rb

```diff
@@ -785,6 +788,25 @@
         end
       end
     end
+
+    describe "skip_session_data" do
+      context "when set to true" do
+        let(:config_options) { { :skip_session_data => true } }
+
+        it "prints a deprecation warning" do
```

Comment:
Would you also add a test, or an assertion to the existing tests, to check that it sets the `send_` config option to the opposite value?